src/style.scss

```diff
@@ -26,6 +31,189 @@ ul {
     height: 100%;
 }
 
+/* MODAL */
+/* Background effect. It covers the entire site */
+.modal {
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Overlay effect: translucent background: black w/ partial opacity */
+    z-index: 1; /* Overlay effect: positioned over other containers */
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    position: fixed; /* Fix position on the top-left corner*/
+    top: 0;
+    left: 0;
+    overflow: auto; /* Enable scroll if needed */
+    padding-top: 80px; /* Location of the content container */
+}
+
+#navBar {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #000000;
+  text-align: right;
+}
+
 .active {
   color: pink;
 }
+
+.flexbox-demo {
+  // display: flex;
+  // justify-content: center;
+  // flex-wrap: wrap;
+  // height: 900px;
+  // flex-direction: row;
+  display: flex;
+  height: 208px;
+  margin-left: 290px;
+  margin-top: 155px;
+  width: 33%;
+  flex-direction: row;
+}
+
+.flexbox-boxes {
+  // display: flex;
+  // justify-content: center;
+  // flex-wrap: wrap;
+  // height: 900px;
+  // flex-direction: row;
+  display: flex;
+  height: 220px;
+  // margin-left: 290px;
+  // margin-top: 155px;
+  width: 100%;
+  flex-direction: column;
+}
+
+.flexbox-second-level {
+  // border: 5px solid black;
+  // background-color: #000000;
+  display: flex;
+  height: 208px;
+  margin-left: auto;
+  margin-top: 155px;
+  width: 508px;
+  flex-direction: row;
+}
+
+.flexbox-second-level:first-child {
+  align-items: flex-end;
+  justify-content: flex-end;
+}
+
+.flexbox-second-level:nth-child(2) {
+  align-items: flex-start;
+  justify-content: center;
+}
+
+.flexbox-second-level:nth-child(3) {
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.flexbox-second-level:last-child {
+  align-items: center;
+  justify-content: center;
+}
+
+.item-black {
+  background-color: #000000;
+  height: 25%;
+  width: 100%;
+}
+
+.item-black:first-child {
+  align-items: flex-end;
+  justify-content: flex-end;
+}
+
+.item-black:last-child {
+  align-items: center;
+  justify-content: center;
+}
+
+.item-white {
+  background-color: #FFFFFF;
+  height: 25%;
+  width: 100%;
+}
+
+.item-white:first-child {
+  align-items: flex-end;
+  justify-content: flex-end;
+}
+
+.item-white:last-child {
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
are these from a tutorial/demo? Should probably remove if you're not using them anymore